ipi/engine/forcefields.py

```diff
@@ -445,11 +446,13 @@ def evaluate(self, r):
         r["result"] = [v, f.reshape(nat * 3), np.zeros((3, 3), float), {"raw": ""}]
         r["status"] = "Done"
 
+
 class FFdmd(ForceField):
 
-    """Basic fully pythonic force provider.
+    """Pythonic force provider.
```

Comment:
Why is this both a smotion class and a FF? It's awkward because now one can also use it as a forcefield outside an actual DMD calculation

Reply:
I don't understand what you would like to see done here. The reason to do it like this is explained. You would like me to code the whole procedure as only the dmd part?

Reply:
If someone wants to use it in some way out of the dmd step I don't see why not actually.